parlai/core/torch_agent.py

```diff
@@ -311,3 +323,26 @@ def shutdown(self):
         if path is not None:
             self.save(path + '.shutdown_state')
         super().shutdown()
+
+    def reset(self):
+        """Reset observation and episode_done."""
+        self.observation = None
+        self.episode_done = True
```

Comment:
Actually I think we want this to look like the seq2seq reset:

https://github.com/facebookresearch/ParlAI/blob/2951872be31f842cea861bcd52c34ed17e247f23/parlai/agents/seq2seq/seq2seq.py#L400-L406

Reply:
See comment w.r.t. observe below.